packages/api/utils/roadworkTypes.zod.ts

```diff
@@ -0,0 +1,98 @@
+import { z } from "zod";
+
+const workStatus = z.union([
+    z.literal("planned"),
+    z.literal("in_progress"),
+    z.literal("completed"),
+    z.literal("cancelled"),
+    z.literal("non_notifiable"),
+    z.literal("unattributable"),
+    z.literal("historical"),
+    z.literal("section_81"),
+]);
+
+export type WorkStatus = z.infer<typeof workStatus>;
+
+export const worksLocationType = [
```

Comment:
I'd maybe call this `worksLocationTypes` since it's an array

Reply:
done